docs-gen/content/rule_set/data_entry/sensor_actuator.md

```diff
@@ -8,57 +8,59 @@ Sensors are signals to read values of properties in a vehicle. Values of sensors
+`description`| Describes the meaning and content of the signal. The `description`shall together with other members like `datatype` and `unit` provide sufficient information to understand what the signal contains and how signal values shall be constructed or interpreted. Recommended to start with a capital letter and end with a dot (`.`).
 
-**```min```** *[optional]*
-The minimum value, within the interval of the given ```datatype```, that the
-data entry can be assigned.
-If omitted, the minimum value will be the "Min" value for the given datatype.
-Cannot be specified if ```allowed``` is defined for the same data entry.
+## Optional Data Entry Attributes
 
-**```max```** *[optional]*
-The maximum value, within the interval of the given ```datatype```, that the
-data entry can be assigned.
-If omitted, the maximum value will be the "Max" value for the given datatype.
-Cannot be specified if ```allowed``` is defined for the same data entry.
+In additon to the mandatory attributes some optional attributes have been defined.
+There may be additional constraints on their usage not specified in the table below.
 
-**```unit```** *[optional]*
-The unit of measurement that the data entry has. See [Data Units](/vehicle_signal_specification/rule_set/data_entry/data_unit_types/)
-chapter for a list of available units. This
-cannot be specified if ```allowed``` is defined for the same data entry.
+Attribute    | Description                 | Comment
+-------------|-----------------------------|--------
+`comment`    | A comment can be used to provide additional informal information on a signal. This could include background information on the rationale for the signal design, references to related signals, standards and similar. Recommended to start with a capital letter and end with a dot (`.`). | *since version 3.0*
+`min`        | The minimum value, within the interval of the given `datatype`, that the data entry can be assigned. If omitted, the minimum value will be the "Min" value for the given datatype. Cannot be specified if `allowed` is defined for the same data entry.
+`max` | The maximum value, within the interval of the given `datatype`, that the data entry can be assigned. If omitted, the maximum value will be the "Max" value for the given datatype. Cannot be specified if `allowed` is defined for the same data entry.
+`unit` | The unit of measurement that the data entry has. See [Data Units](/vehicle_signal_specification/rule_set/data_entry/data_units/) chapter for a list of available units. Cannot be specified if `allowed` is defined for the same data entry.
+`pattern` | Can be used for datatype `string` to specify a regular expression that limits allowed values for the data entry. The expression must be supported by [Python Regular Expressions](https://docs.python.org/3/howto/regex.html) | *since version 6.0*
```

Comment:
Would it worth if we say that patterns can be used for `string` and `string[]` or it can be assumed that after one type is allowed, then its plural (array) version is also allowed?

Other than this, I checked the online test link, you provided: https://erikbosch.github.io/vehicle_signal_specification/rule_set/data_entry/sensor_actuator/index.html and this part is OK for me.

Reply:
Makes sense to mention also `[]`, will update